Tangem/Modules/OrganizeTokens/ListHeader/OrganizeTokensHeaderView.swift

```diff
@@ -12,29 +12,34 @@ struct OrganizeTokensHeaderView: View {
     @ObservedObject var viewModel: OrganizeTokensHeaderViewModel
 
     var body: some View {
+        let shadowOpacity = 0.1
+        let toggledShadowOpacity = shadowOpacity / (viewModel.isSortByBalanceEnabled ? 3.0 : 1.0)
```

Comment:
```suggestion
var toggledShadowOpacity: CGFloat { 
    let shadowOpacity = 0.1
        let toggledShadowOpacity = shadowOpacity / (viewModel.isSortByBalanceEnabled ? 3.0 : 1.0)
        return toggledShadowOpacity
}
    var body: some View {}
```

Может отдельно рассчитываемой ?

Reply:
Логично, вынес
Ну и нейминг поправил, если это значение расчитывается на основе конкретного параметра `isSortByBalanceEnabled` - то оно должно называться вовсе не дженерик `toggled*`